point-addition/src/conversion.rs

```diff
@@ -0,0 +1,91 @@
+//! This module implements the conversion from the sum of two elliptic curve points to the sum of
```

Comment:
I dont think the word "conversion" is fitting for this module.

E.g. M2A means "multiplicative shares to additive shares" conversion. Multiplicative shares is what we have in the beginning of the protocol. 

If we say "conversion from the sum" that would mean that "sum" is sth that was originally available, which is not accurate.

Instead I suggest to use the wording similar to what is used in lib.rs

//! This module implements a secure two-party computation protocol for adding two private EC points and secret-sharing the resulting x coordinate (the shares are field elements of the field underlying the elliptic curve).
//! This protocol has semi-honest security.